events/javaland/info.html.slim

```diff
@@ -0,0 +1,54 @@
+---
+layout: microsite-multi-page
+hero_class: event-javaland
+
+nav: 
+  - title: About JavaLand
+    url: '../about/'
+  - title: Our Speakers
+    url: '../speakers/'
+  - title: Our Sessions
+    url: '../sessions/'
+  - title: Further Information
+    url: '../info/'
+---
+
+- page.title = "JBoss at JavaLand - more Information"
+
+= define 'hero'
+  .row 
+    .large-25.column.center
+      h2 JBoss at JavaLand
+
+h2.divider JavaLand - More Information
+p  Red Hat will be on site with our own booth and we will provide room for all kinds of get-togethers. If you're staying in one of the park hotels, you'll be surprised with a little present and it will be our pleasure to provide charging stations at our booth.  
+
+p  Make sure to catch up with all the Red-Hatters on site, as we know that there'll be even more than just our speakers. 
+
+asciidoc:  
+   And if you want to start working with Red Hat JBoss Middleware we suggest to start trying out JBoss Enterprise Application Platform. There is plenty of information available link:http://www.jboss.org/enterprise-java[on JBoss.org ] and you can also directly download the link:http://www.jboss.org/products/eap/overview/[latest supported release].
+
+p: strong Beta Release
+p Featuring many updates, including targeted support of Java 8 and updated database certifications. Try it today!
+
+a.button(href="http://www.jboss.org/download-manager/file/jboss-eap-6.4.0.Beta.zip") JBoss EAP 6.4.0.Beta
```

Comment:
Do we really want to use the CTA for downloading a Beta rather than a GA? I don't know what the strategy is here, so just making sure this is the right choice.